packages/angular/src/generators/application/application.ts

```diff
@@ -60,6 +60,7 @@ export async function applicationGenerator(
 
   await angularInitGenerator(tree, {
     ...options,
+    tsConfigName: schema.rootProject ? 'tsconfig.json' : 'tsconfig.base.json',
```

Comment:
I prefer this methodology for passing this. It doesn't appear on the `ApplicationSchema`